mlocal/checks/basechecks.chk

```diff
@@ -79,6 +79,10 @@ if [ $do_go_version_check = 0 ]; then
 	fi
 elif [ "$hstgo" = "" ]; then
 	printf " checking: host Go compiler (at least version $hstgo_version)... "
+	if [ "$hstgo_opts" = "" ]; then
+		echo "\$hstgo_opts is undefined! Cannot check host Go compiler."
+		exit 1
+	fi
```

Comment:
Rather than adding this, let's delete hstgo_opts from mconfig and just use "go" instead of the for loop and $go below.  It's unnecessarily obscure.